### DISCUSSION TITLE
基座如果无法获取到子应用，如何监听错误事件并做出动作

### DISCUSSION THREAD

**Discussion by 826327700** [jd-opensource/micro-app#131] [Q&A]
使用场景：基座跳转到/app1，恰好app1的应用无法连接上（如服务器宕机），这时候，希望基座可以通过某个回调，跳转到友好的提示页面。

**Comment by bailicangdu** [comment]
如果是页面加载出错，可以通过error事件监听 https://micro-zoe.github.io/micro-app/docs.html#/zh-cn/life-cycles?id=_5-error

**Comment by bailicangdu** [comment]
如果是子应用内部报错，目前无法监听
